### DISCUSSION TITLE
Prevent browser-side logs(console.error, console.warning, console.error, etc) from showing up on the terminal.

### DISCUSSION THREAD

**Discussion by Progressandro** [vercel/next.js#16355] [Help]
As title states, I would like to check out browser logs from appearing on the terminal, **so it can show just server-side logs** (logs made in getServerSideProps, API routes, etc). Is there a config option to achieve this without much tinkering?

Thanks in advance.

**Comment by timneutkens** [comment]
logs created in the browser are not piped to the terminal in any way.
